samples/katib/early-stopping.ipynb

```diff
@@ -201,6 +201,7 @@
     "# Update the PIP version.\n",
     "!python -m pip install --upgrade pip\n",
     "!pip install kfp==1.1.1\n",
+    "!pip install kfp-tekton==0.4.0\n",
```

Comment:
maybe put it under `kubeflow-katib`, I realized kubeflow-katib is using kubernetes==10.0.1 which is breaking the kfp client package.